content/en/blog/falco-okta-identity/index.md

```diff
@@ -0,0 +1,168 @@
+```
+
+Here is the output displayed in our Falcosidekick UI, presenting the context of the actor's name and the corresponding IP address responsible for the change. It could be advantageous to explore the possibility of incorporating additional ‘Output’ fields to enhance incident response capabilities when addressing identity threats.
+
+
+![](https://github.com/falcosecurity/falco-website/tree/master/content/en//blog/falco-okta-identity/images/okta4.png)
+
+## Building a custom Falco rule
+The next example extends beyond the scope of the default Falco rules. For instance, if an identity integration or application were to be disassociated from the user 'Nigel Douglas,' it might be an attempt to compromise security measures within established workflows or systems - a good example of ‘Impair Defenses’ technique in the MITRE ATT&CK matrix. 
+
+Consequently, we will replicate this specific action and create a custom Falco rule to identify such behavior. As shown in the screenshot below, the admin user is seen removing the application assignment from the user Nigel.
+
+![](https://github.com/falcosecurity/falco-website/tree/master/content/en//blog/falco-okta-identity/images/okta5.png)
+
+![](https://github.com/falcosecurity/falco-website/tree/master/content/en//blog/falco-okta-identity/images/okta6.png)
+
+After the application has been unassigned, we receive the updated event type data in Okta, much like our previous workflow. The Okta query that provides the results of the application removal is as follows:
+
+```
+eventType eq "application.user_membership.remove"
+```
+
+![](https://github.com/falcosecurity/falco-website/tree/master/content/en//blog/falco-okta-identity/images/okta7.png)
+
+With this understanding, we can navigate to the ([falco_rules.local.yaml](https://falco.org/docs/rules/default-custom/#the-configuration-file)) file linked to our Falco installation, which serves as a local repository for our Falco rules. Having grasped the construction of the previous Falco rule, we will substitute 'okta.evt.type' for the Okta attribute 'eventType' and assign it the precise string identified in the screenshot above. In your case, the Falco rule should be structured as follows:
+
+```
+- rule: Remove app membership
+  desc: Detect membership removal in OKTA
+  condition: okta.evt.type = "application.user_membership.remove"
+  output: "A user has removed the following app in OKTA (user=%okta.actor.name, ip=%okta.client.ip, target user=%okta.target.user.name)"
+  priority: CRITICAL
+  source: okta
+  tags: [custom_rule, mitre_defense_evasion, T1562, impair_defenses]
+```
+
+To ensure the custom rule is applied, it is necessary to restart the Falco service. If you have deployed Falco and Falcosidekick via a Docker compose file, this can be achieved by simply executing a 'stop' command to halt the containers, followed by the 'up -d' command, which restarts the Docker containers with the same configurations specified in the docker-compose.yaml file. 
```

Comment:
```suggestion
To ensure the custom rule is applied, it is necessary to restart the Falco service. If you have deployed Falco and Falcosidekick via a Docker Compose file, this can be achieved by simply executing a `stop` command to halt the containers, followed by the `up -d` command, which restarts the Docker containers with the same configurations specified in the docker-compose.yaml file. 
```